searxng_extra/update/update_firefox_version.py

```diff
@@ -64,6 +64,12 @@ def fetch_firefox_last_versions():
     major_last = versions[0].major
     major_list = (major_last, major_last - 1)
     for version in versions:
+        msg = (
+            "Please check if the rv segment of the user agent is still frozen at 109.0: "
+            "https://bugzilla.mozilla.org/show_bug.cgi?id=1805967"
+        )
+        assert version.major != 120, msg
```

Comment:
@dalf I'm not familiar with github .. will we take notice when there is an exception in the CI workflow .. or will we silently get none update of the version?

```
Traceback (most recent call last):
  File "searxng_extra/update/update_firefox_version.py", line 88, in <module>
    useragents["versions"] = fetch_firefox_last_versions()
  File "searxng_extra/update/update_firefox_version.py", line 71, in fetch_firefox_last_versions
    assert version.major != 118, msg
AssertionError: Please check if the rv segment of the user agent is still frozen at 109.0: https://bugzilla.mozilla.org/show_bug.cgi?id=1805967
```

Reply:
The CI will fail as it has already happened on the traits (or more often on the language updates).

https://docs.github.com/en/enterprise-server@3.8/actions/creating-actions/setting-exit-codes-for-actions

Reply:
> The CI will fail as it has already happened on the traits (or more often on the language updates).

Ooops I didn't aware that I have to look regularly into github's  "Actions" tab:

![image](https://github.com/searxng/searxng/assets/554536/b96e4950-a040-4b45-9bfe-a90cc9445626)

@dalf: thanks for pointing out :+1: / I'm not a github expert .. if you notice something wrong again, please send me a short note again / thanks!

Reply:
> Ooops I didn't aware that I have to look regularly into github's "Actions" tab:

You should be able to see the failing workflow in your [GitHub notifications](https://github.com/notifications).

You see your notifications from command line using:
* https://cli.github.com/
* and this extension: https://github.com/meiji163/gh-notify

Reply:
> You should be able to see the failing workflow in your [GitHub notifications](https://github.com/notifications).

Seems not to work .. IDK .. I activated some more stuff in my gh settings ..  not sure it helps / but thanks for the tip :+1: